editoast/src/models/track_section_link_layer.rs

```diff
@@ -94,7 +95,7 @@ impl TrackSectionLinkLayer {
         infra: i32,
         operations: &Vec<OperationResult>,
         infra_cache: &InfraCache,
-        invalid_zone: &InvalidationZone,
+        invalidation_zone: &InvalidationZone,
```

Comment:
I was keeping it short to oneline the `invalidate_bbox_cahrtos_layer` function call